src-docs/src/views/markdown_editor/mardown_format_example.js

```diff
@@ -51,7 +55,7 @@ export const MarkdownFormatExample = {
         <p>
           <strong>EuiMarkdownFormat</strong> is a wrapper that will render
           Markdown provided. EuiMarkdownFormat uses{' '}
-          <Link to="https://github.com/remarkjs/remark)">Remark</Link> by
+          <EuiLink href="https://github.com/remarkjs/remark">Remark</EuiLink> by
```

Comment:
Adding the `external` prop will add a little icon next to the link, open the link in a new tab, and add some screen reader only instructions to that effect as well.

All of which we want for links that point outside of EUI

```suggestion
          <EuiLink external href="https://github.com/remarkjs/remark">Remark</EuiLink> by
```